src/test/java/ibm/jceplus/junit/openjceplusfips/TestRSA.java

```diff
@@ -8,11 +8,17 @@
 
 package ibm.jceplus.junit.openjceplusfips;
 
+import java.security.NoSuchAlgorithmException;
+
+import javax.crypto.Cipher;
+
+import ibm.jceplus.junit.base.BaseTest;
 import junit.framework.Test;
 import junit.framework.TestSuite;
 
-public class TestRSA extends ibm.jceplus.junit.base.BaseTestRSA {
+public class TestRSA extends BaseTest {
```

Comment:
Not sure why this changed...

Reply:
Never mind. I got that you changed it to avoid running the tests included in `BaseTestRSA`.